packages/flutter/lib/src/painting/oval_border.dart

```diff
@@ -0,0 +1,54 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' as ui show lerpDouble;
+
+import 'package:flutter/foundation.dart';
+
+import 'borders.dart';
+import 'circle_border.dart';
+
+/// A border that fits an elliptical shape.
+///
+/// Typically used with [ShapeDecoration] to draw an oval.
+/// Instead of centering the [Border] to a square, like [CircleBorder],
+/// it fills the available space, such that it touches the edges of the box.
+/// There is no difference between `CircleBorder(eccentricity = 1.0)` and `OvalBorder()`.
+/// [OvalBorder] works as an alias for users to discover this feature.
```

Comment:
You should fill and wrap this at 80 columns:
```suggestion
/// Typically used with [ShapeDecoration] to draw an oval. Instead of centering
/// the [Border] to a square, like [CircleBorder], it fills the available space,
/// such that it touches the edges of the box. There is no difference between
/// `CircleBorder(eccentricity = 1.0)` and `OvalBorder()`. [OvalBorder] works as
/// an alias for users to discover this feature.
```